### ISSUE TITLE
Which layers of the model are finetuned?

### ISSUE THREAD

**Issue by ManjeeraJagiri** [experiencor/keras-yolo3#72]
I understand when we set 'by_name=True', only those layers with same names will be loaded with weights and the rest finetuned. But can I know which section of layers exactly in yolo.py will be finetuned? And where are we initializing the weights of these layers? Thanks!
```
    if os.path.exists(saved_weights_name): 
        print("\nLoading pretrained weights.\n")
        template_model.load_weights(saved_weights_name)
    else:
        template_model.load_weights("backend.h5", by_name=True)
```

**Comment by yuyijie1995** [comment]
Hello，have you figure out  this problem?

**Comment by ManjeeraJagiri** [comment]
@yuyijie1995 , Nope. I haven't

**Comment by XiaoSX** [comment]
I am inspecting the backend.h5 file, found 249 variables of yolo inside the h5 file. only 10 variables are missed, they are ['conv_81', 'input_3', 'input_2', 'conv_93', 'input_4', 'conv_105', 'input_5', 'yolo_layer_1', 'yolo_layer_2', 'yolo_layer_3']. besides the input layer(for input) and yolo layer(for caculating loss). only conv_81, _93, _105 are missed, they are the last conv layers which have the specific num_classes outputs, maybe only finetune these layers
